.github/workflows/build_doc_ci.yml

```diff
@@ -18,7 +18,32 @@ env:
   testWorkingDirectory: ${{ github.workspace }}/src/promptflow
 
 jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+    - name: Python Setup
+      uses: "./.github/actions/step_create_python_environment"
+    - name: Build wheel
+      uses: "./.github/actions/step_sdk_setup"
+      with:
+        setupType: ${{ env.packageSetupType }}
+        scriptPath: ${{ env.testWorkingDirectory }}
+    - name: Upload Wheel
+      if: always()
+      uses: actions/upload-artifact@v3
+      with:
+        name: wheel
+        path: |
+          ${{ github.workspace }}/src/promptflow/dist/*.whl
+          ${{ github.workspace }}/src/promptflow-tools/dist/*.whl
+          ${{ github.workspace }}/src/promptflow-tracing/dist/*.whl
```

Comment:
why this step?
pip install -e will do the work for you.

Reply:
Refined.